frontend/package.json

```diff
@@ -3,17 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@grpc/proto-loader": "^0.3.0",
     "@material-ui/core": "3.7.1",
     "@material-ui/icons": "^3.0.1",
     "@types/js-yaml": "^3.11.2",
     "brace": "^0.11.1",
     "d3": "^5.7.0",
     "d3-dsv": "^1.0.10",
     "dagre": "^0.8.2",
+    "frontend": "kubeflow/frontend#756476a",
```

Comment:
It shouldn't be too much effort but it will require a change in kubeflow/frontend. I've filed an issue to move the frontend package under a scope. I'd like to get to that after this PR so we don't have to wait for the package.json change in that repo.

https://github.com/kubeflow/frontend/issues/24

Reply:
Actually, just updating package.json works correctly, so I've made this change now.